scripts/core/editor3/actions/editor3.jsx

```diff
@@ -0,0 +1,21 @@
+export function onChangeEditorState(editorState) {
```

Comment:
Actions can be called from entire application so it is more clear to have onChangeEditorState. OnChange is too general and also other components can have it.